tools/wpt/browser.py

```diff
@@ -635,8 +635,11 @@ def install(self, dest=None, channel=None):
     def find_binary(self, venv_path=None, channel=None):
         raise NotImplementedError
 
-    def find_webdriver(self):
-        return find_executable("safaridriver")
+    def find_webdriver(self, channel=None):
+        path = None
+        if channel == "preview":
+            path = "/Applications/Safari Technology Preview.app/Contents/MacOS"
```

Comment:
Bleh. How sure are we that this is the only possible path?

Reply:
It's where the dmg is installed and the installer doesn't give any option to put it elsewhere. I think you could well install it elsewhere at the command line. But then you'll have to use `--webdriver-binary`.

Reply:
OK.